eslint/cactbot-output-strings.js

```diff
@@ -54,8 +57,36 @@ const ruleModule = {
       return propKeys;
     };
 
+    const extractTemplate = function(node) {
+      if (node.properties === undefined) return;
+      const outputTemplateKey = {};
+      for (const outputString of node.properties.filter((s) => s.type !== 'SpreadElement' && s.value.type !== 'MemberExpression')) {
+        // each outputString
+        const values = [];
+
+        outputString.value.properties.forEach((x) => {
+          if (x.value.value !== undefined)
+            values.push(x.value.value);
+        });
+        const templateIDs = values.map((x) => Array.from(x.matchAll(/\${\s*([^}\s]+)\s*}/g))).map((x) => x.length ? x.map((v) => v[1]) : null);
```

Comment:
This is just a subjective opinion, but can you name anything with an `id` suffix as `Id` and not `ID`, e.g. `templateId` to be consistent with other uses in cactbot and even in this file (e.g. `messageId`)?